static/docs/commands-reference/get.md

```diff
@@ -0,0 +1,45 @@
+# get
+
+Download or copy file or directory from another DVC repository into the local
+file system.
+
+> Unlike `dvc import`, this command does not track the downloaded data file(s)
+> (does not create a DVC-file).
+
+## Synopsis
+
+```usage
+usage: dvc get [-h] [-q | -v] [-o [OUT]] [--rev [REV]] url path
+
+positional arguments:
+  url                   DVC repository URL to download data from.
+  path                  Path to data within DVC repository.
+```
+
+## Description
+
+In some cases it's convenient to get a <abbr>data artifact</abbr> from another
+DVC repository. The `dvc get` command helps the user do so. The `url` argument
+should provide the external DVC project's Git repository URL, while `path` is
```

Comment:
`DVC project's Git repository URL` - this is kind of a clarification I was mentioning above

Reply:
Yes. I added a note that both HTTP and SSH [protocols](https://git-scm.com/book/en/v2/Git-on-the-Server-The-Protocols) are supported in 50f7c6e. Please check.

Reply:
👍 also, see the version of the phrase above

Reply:
I didn't understand your last comment ): What version? What phrase?

If you're referring to a previous comment in this PR, a lot of them I've resolved and even so this PR is gotten so long that almost everything is hidden by default when the page loads. So please repeat the comment in the code if needed, thanks!

Reply:
Ha :) It's hard to track things for me as well. I probably meant `URL of Git repository with DVC project to download from`